packages/hooks/src/useCollectionQuery/__mocks__/queries.ts

```diff
@@ -0,0 +1,72 @@
+import gql from "graphql-tag";
```

Comment:
This can be imported from Apollo.

```suggestion
import { gql } from "@apollo/client";
```